spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -205,14 +205,22 @@
   end
 
   describe "#to_json" do
-    subject { invenio_rdm_record_converter.to_json }
+    context "record is exportable" do
+      subject { invenio_rdm_record_converter.to_json }
 
-    it do
-      is_expected.to eq json
+      it "returns formatted json string" do
+        is_expected.to eq json
+      end
+    end
+
+    context "record is unexportable" do
```

Comment:
Ensure that unexportable records return the blank json we're expecting and not something like "null"